apps/src/dance/Dance.js

```diff
@@ -680,6 +680,9 @@ Dance.prototype.displayFeedback_ = function() {
     artistTwitterHandle +
     ' on @codeorg!';
 
+  const comma = '%2C';
+  const hashtags = ['codeplay', 'HourOfCode'].join(comma);
```

Comment:
I think we should move these hashtags to someplace like dance/constants.js so that in the future we could updated a single location rather than here and ShareAllowedDialog. It might make sense to move the comma const there as well, just for simplicity's sake.

Reply:
Absolutely agreed that the need to do this twice is a sign that some refactoring should be done.  

That said, I think the right refactoring is a bit larger in scope:

- the share code shouldn't need to know about specific tutorial families.  Rather than having share code import from dance code, the share code should be agnostic about the type of tutorial it's used for, and so things that vary between tutorials, such as hashtags, artist handles, etc. should be passed in from the tutorial-specific code.
- there shouldn't be two separate implementations of share UI.

I'm open to a better factoring of this code, but I'm not sure it's going to solve the broader issues, and want to keep this change as small as possible.

Reply:
Some of this potential refactoring was mentioned [here](https://github.com/code-dot-org/code-dot-org/pull/26244#issuecomment-442528103).

Reply:
It makes me 'oof' when I read good refactoring ideas from 2018 that are still timely. Would you take create a JIRA task to track this refactoring tech debt? The fact that we ran back into it makes me feel like we should pay it down. And I agree that it's out of scope for this change.